src/script/components/Modals/UserModal/UserModal.styles.ts

```diff
@@ -50,11 +50,16 @@ export const userModalStyle = css`
   }
 
   .modal__body {
-    padding: 16px;
+    padding: 0 16px;
 
     .classified-bar {
       margin-top: 12px;
       width: calc(100% + 32px);
     }
   }
 `;
+
+export const userModalWrapperStyle: CSSObject = {
+  maxHeight: '90vh',
```

Comment:
an other option would be `fit-content`, `90vh` allows makes it more obvious the component is a modal on smaller windows
![image](https://github.com/wireapp/wire-webapp/assets/78490891/56311bfe-6776-4615-9ac8-22cebf1fb75c)